amplifyconfiguration.json -> amplify_outputs.json (#7346)
aws-amplify/docs: src/pages/gen2/build-a-backend/server-side-rendering/index.mdx

You will need to create a `runWithAmplifyServerContextRunner` function to use Amplify APIs on the server-side of your Next.js app.

> You can create an `amplifyServerUtils.ts` file under a `utils` folder in your codebase. In this file, you will import the Amplify configuration object from the `amplify_outputs.json` file that is generated by the Amplify CLI, and use the `createServerRunner` function to create the `runWithAmplifyServerContextRunner` function.

Reviewer: ```suggestion
You can create an `amplifyServerUtils.ts` file under a `utils` folder in your codebase. In this file, you will import the Amplify backend outputs from the `amplify_outputs.json` file that is generated by the Amplify CLI, and use the `createServerRunner` function to create the `runWithAmplifyServerContextRunner` function.
```

Is `Amplify CLI` the right wording here?

Author: > Is Amplify CLI the right wording here?

it is not

For example, the `utils/amplifyServerUtils.ts` file may contain the following content:

```typescript
import { createServerRunner } from '@aws-amplify/adapter-nextjs';
import outputs from '@/amplify_outputs.json';

export const { runWithAmplifyServerContext } = createServerRunner({

>   config

Reviewer 2: ```suggestion
  outputs
```

```typescript
'use client';

import outputs from '@/amplify_outputs.json';
import { Amplify } from 'aws-amplify';

> Amplify.configure(config, {

Reviewer 2: ```suggestion
Amplify.configure(outputs, {
```

'use client';

import { Amplify } from 'aws-amplify';
import outputs from '../amplify_outputs.json';

> Amplify.configure(config, { ssr: true });

Reviewer 2: ```suggestion
Amplify.configure(outputs, { ssr: true });
```
